src/value-converters/abstract-locale-value-converter.ts

```diff
@@ -18,7 +18,7 @@ export abstract class AbstractLocaleValueConverter {
      * Returns the instance or system locale
      */
     getLocale() {
-        return this._locale ? this._locale : AbstractLocaleValueConverter.getSystemLocale();
+        return this._locale ? this._locale : AbstractLocaleValueConverter._systemLocale;
```

Comment:
If we don't do this, `AbstractLocaleValueConverter._systemLocale` is never read, raising [noUnusedLocals](https://www.typescriptlang.org/tsconfig#noUnusedLocals).

Reply:
I'm not sure why this field exists, since we don't have tests for that. Maybe we should just remove it completely.

Reply:
Done.